### DISCUSSION TITLE
Tile card: Make shaping optionally

### DISCUSSION THREAD

**Discussion by KNXBroker** [home-assistant/frontend#14889] [Dashboard]
I love the new tile card, however not the (mushroom) shaping. For me the shaping makes the icon hard to read (blurry) and an eye-catcher (which I do not want to have), especially in dark themes. Furthermore clicking more-info does also not contain any shaping. 

Can you make the shaping optionally? This would also give the possibility to be consistent with the old (existing) cards, more-info, etc.

Example with shaping active:
![image](https://user-images.githubusercontent.com/25910703/209560195-f6f5d17b-5166-4ba7-a2ef-833848d052b2.png)

Example with shaping inactive:
![image](https://user-images.githubusercontent.com/25910703/209560249-36afd9e9-0e84-4372-9b7d-356f9a4c5412.png)

**Comment by ildar170975** [comment]
I agree that the new Tile card gives a lot of features - but it does look differently if compared with other cards. I mean icons with a background. 
Probably all conventional (not custom) cards must look similarly. 

At first glance I thought that a possible solution could be using a theme variable for an icon’s background.
But a bit later I realized that this background depends on the current icon’s color, so it is not just some theme variable. 
Now I think that an option like “disable_icon_background” for the Tile card (“false” by default) could be a way to remove this background - for users who do not like it. 
Surely, also there is still a way of using card-mod to remove a background)))

**Comment by KNXBroker** [comment]
Another idea: making the opacity value a theme variable...
